tests/integrations/basic-fail/Cargo.stderr

```diff
@@ -37,6 +38,17 @@ For more information about this error, try `rustc --explain E0308`.
 
 
 
+tests/actual_tests/exit_code_fail.rs FAILED:
+command: "rustc" "--error-format=json" "--out-dir" "/tmp/.tmpp2eImK" "--edition=2021" "--extern" "basic_fail=$DIR/$DIR/../../../target/debug/libbasic_fail-$HASH.rmeta" "-L" "$DIR/$DIR/../../../target/debug" "-L" "$DIR/$DIR/../../../target/debug" "tests/actual_tests/exit_code_fail.rs"
+
+fail test got exit status: 0, but expected 1
+
+no error patterns found in failure test
```

Comment:
So it's 'failure test' once and 'fail test' the other time. We should make that consistent.